docs/services/ml/_index.md

```diff
@@ -28,7 +28,12 @@ For some models, like the [Triton MLModel](/modular-resources/examples/triton/)
 
 {{<modular-resources api="rdk:service:mlmodel" type="mlmodel">}}
 
-{{< readfile "/static/include/create-your-own-mr.md" >}}
+{{< alert title="Add support for other models" color="tip" >}}
+If none of the existing models fit your use case, you can create a {{< glossary_tooltip term_id="modular-resource" text="modular resource" >}} to add support for it.
+
+ML Models must be designed in particular shapes to work with Viam's [vision services](/services/vision/).
```

Comment:
Not sure if you wanna do this one but it's more accurate to say 
"... to work with Viam's vision service." --> "... to work with the mlvision model of Viam's vision service."
cuz no other type of vision service is gonna even try

Reply:
Good point! Added to intro on /mlmodel-design page as well